build-system/pr-check/checks.js

```diff
@@ -78,6 +79,7 @@ async function prBuildWorkflow() {
   // Check document links only for PR builds.
   if (buildTargetsInclude(Targets.DOCS)) {
     timedExecOrDie('gulp check-links --local_changes');
+    timedExecOrDie('gulp markdown-toc');
```

Comment:
In addition to running this check when a document file is modified, we should run it when the `markdown-toc` directory is modified in any way. To do this, add this additional case...

```js
file.startsWith('build-system/tasks/markdown-toc/') ||
```

... to this matcher function...

https://github.com/ampproject/amphtml/blob/2acbd7db88491ca43b49364de4610f235a09f26b/build-system/pr-check/build-targets.js#L167-L175